scripts/commit_classifier.py

```diff
@@ -136,6 +144,12 @@ def __init__(self, model_name, repo_dir, git_repo_dir, method_defect_predictor_d
         if git_repo_dir:
             self.clone_git_repo("https://github.com/mozilla/gecko-dev", git_repo_dir)
 
+        repository.clone(self.repo_dir)
+
+        with hglib.open(self.repo_dir) as hg:
+            if phabricator_deployment is not None and diff_id is not None:
+                self.apply_phab(hg, phabricator_deployment, diff_id)
+
```

Comment:
We could move the `download_and_load_model` after this, so we don't download the models needlessly if we can't apply the patch.

Reply:
Yeah, makes sense.